### PR TITLE
Add News views

### PR DESCRIPTION
**Pull request by PabloMM2**
Noticias index, show y formulario ejemplo

### CODE REVIEW

File: public/css/manager.css [hunk 1]
```diff
@@ -11741,3 +11741,45 @@ body {
           box-shadow: 0 0 0 3px rgba(255, 255, 255, 0.25);
 }
 
+.news-card{
+    margin: 20px 0;
+}
+
+.news-card2{
```
Comment (dsalcedo) [inline public/css/manager.css]: 🤔 Why not use `p2 or p5`  or `pt-2, pb-1`?
Look in boostrap https://getbootstrap.com/docs/4.1/utilities/spacing/

Same usage case for news-card

File: public/css/manager.css [hunk 2]
```diff
@@ -11741,3 +11741,45 @@ body {
...
+  padding:15px 20px; 
+}
+
+.news-head{
```
Comment (dsalcedo) [inline public/css/manager.css:11745]: If you are using `.card` css can use te `text-*` helper

```
<div class="card">
  <div class="card-header">
    Featured
  </div>
  <div class="card-body">
    <h5 class="card-title text-center">Special title treatment</h5>
    <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
    <a href="#" class="btn btn-primary">Go somewhere</a>
  </div>
</div>
```

File: public/css/manager.css [hunk 3]
```diff
@@ -11741,3 +11741,45 @@ body {
...
+  display: flex; 
+  align-items: center; 
+}
+
+.news-option{
+  margin: 0 10px;
+}
+
+.news-span{
```
Comment (dsalcedo) [inline public/css/manager.css:11750]: 🤔 Whats the difference with https://getbootstrap.com/docs/4.1/components/badge/ ??

File: public/css/manager.css [hunk 4]
```diff
@@ -11741,3 +11741,45 @@ body {
...
+  height: 25px;
+  width: 25px;
+}
+
+.perfil-image{
```
Comment (dsalcedo) [inline public/css/manager.css]: With Bootstrap can get thumb css class

https://getbootstrap.com/docs/4.1/content/images/#image-thumbnails

Only we need is add the `.avatar` width and height

File: public/css/manager.css [hunk 5]
```diff
@@ -11741,3 +11741,45 @@ body {
...
+  width:70px; height:70px; border-radius:150px;border:1px solid #666;
+}
+
+.news-item{
```
Comment (dsalcedo) [inline public/css/manager.css]: Look in boostrap https://getbootstrap.com/docs/4.1/utilities/spacing/

File: public/css/manager.css [hunk 6]
```diff
@@ -11741,3 +11741,45 @@ body {
...
+  margin: 10px 10px;
+}
+
+.carousel-size{
+  height: 400px;
+}
+
+.news-title{
```
Comment (dsalcedo) [inline public/css/manager.css]: change to `font-arial-black` class name and apply in html

File: public/css/manager.css [hunk 7]
```diff
@@ -11741,3 +11741,45 @@ body {
...
+  font-family: "Arial Black", Gadget, sans-serif;
+}
+
+.news-contenido {
```
Comment (dsalcedo) [inline public/css/manager.css]: Same here Look in boostrap https://getbootstrap.com/docs/4.1/utilities/spacing/
if are you using new font-family create `font-*` class name in global CSS

File: resources/views/manager/news/create.blade.php [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+        <h1 class="h2">Creando Noticia</h1>
+    </div>
+    <div class="container-fluid">
+        <div class="col-md-12">
+            <div class="row">
+                <div class="col-md-1">
+                   
+                </div>
+
+                <div class="col-md-10" style="padding: 15px; border: 0.5px solid black; border-radius: 30px;">
```
Comment (dsalcedo) [inline resources/views/manager/news/create.blade.php]: use https://getbootstrap.com/docs/4.1/utilities/spacing/ instance of  `padding: 15px; ` lets move all the "style" to custom class names

File: resources/views/manager/news/create.blade.php [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+                         <textarea class="form-control" name="contenido" aria-label="With textarea" rows="6" ></textarea>
+                      </div>
+                      <br>
+                      <div class="input-group">
+                        <button class="btn btn-outline-success left">Publicar noticia</button>
+                      </div>
+                    </div>
+                  </form>
+                </div>
+                <div class="col-md-1">
```
Comment (dsalcedo) [inline resources/views/manager/news/create.blade.php]: remove col-md-1 if not used

**Review by ghost — COMMENTED** [review]
Todo bn :3

File: resources/views/manager/news/show.blade.php [hunk 10]
```diff
@@ -46,24 +44,22 @@
...
                           </div>
                     </div>
-                    <div class="col-md-12 news-item">
-                      <p class="news-contenido">
+                    <div class="col-md-12  news-item py-3 px-3">
+                      <p class="font-times-new-roman">
                         "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
                       </p>
-                      <p class="news-contenido">
+                      <p class="font-times-new-roman">
```
Comment (dsalcedo) [inline resources/views/manager/news/show.blade.php:51]: I think if in ‘news-item’ the font family should be times news roman,  this font class name needs move to the parent div. Example:

```html
<div class=“news-item font-times-new-roman ...”>
```

File: public/css/manager.css [hunk 11]
```diff
@@ -11741,45 +11741,31 @@ body {
...
-.news-item{
-  margin: 10px 10px;
-}
-
 .carousel-size{
   height: 400px;
 }
 
-.news-title{
+.create-news-container{
```
Comment (dsalcedo) [inline public/css/manager.css:11759]: all the css class should be in the SASS file, if have any questions please contact me 🤓

**Pull request closed without merging**